portfolio/pom.xml

```diff
@@ -35,6 +35,12 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
```

Comment:
Isn't this duplicated above?

Reply:
Yes! That was an accident - it's now removed.